Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/API/UI/FeedbackForm.hs

```diff
@@ -0,0 +1,50 @@
+{-
+ Copyright 2022-23, Juspay India Pvt Ltd
+
+ This program is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General Public License
+
+ as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version. This program
+
+ is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+
+ or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more details. You should have received a copy of
+
+ the GNU Affero General Public License along with this program. If not, see <https://www.gnu.org/licenses/>.
+-}
+
+module API.UI.FeedbackForm
+  ( API,
+    handler,
+  )
+where
+
+import Domain.Action.UI.FeedbackForm
+import Domain.Types.FeedbackForm
+import qualified Domain.Types.Ride as R
+import Environment
+import Kernel.Prelude
+import Kernel.Types.APISuccess
+import Kernel.Types.Id
+import Kernel.Utils.Common
+import Kernel.Utils.SlidingWindowLimiter
+import Servant
+
+type API =
+  "beckn"
+    :> "booking"
+    :> ( "feedback"
+           :> ReqBody '[JSON] FeedbackFormReq
+           :> Post '[JSON] APISuccess
+       )
+
+handler :: FlowServer API
+handler = getFeedbackFormResult
+
+getFeedbackFormResult :: FeedbackFormReq -> FlowHandler APISuccess
+getFeedbackFormResult req = withFlowHandlerAPI $ do
+  feedbackSubmitApiRateLimitOptions <- asks (.apiRateLimitOptions)
```

Comment:
Do we need to rate limit the feedbackAPI?

Reply:
Yes.
cc: @RitikaHanish

Reply:
@RitikaHanish Why is so?